CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.2.0)
 if(REALM_PLATFORM STREQUAL "Android")
     # This must be before project()
     set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/CMake/android.toolchain.cmake")
-    set(ANDROID_ABI "x86_64" CACHE STRING "")
+    set(ANDROID_ABI "x86" CACHE STRING "")
```

Comment:
Rather than editing this file, do `-DANDROID_ABI=x86` when configuring the project via `cmake`.